File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/struct/p7.c
#include<stdio.h>
#pragma pack(2)
struct Demo{
        char ch1;
        int x;
        float y;
        double arr[8];


};
void main(){
        struct Demo obj;
        printf("%ld\n",sizeof(obj));
        printf("%ld\n",sizeof(struct Demo));
}

File: java-main/java-main/DSA/Recursion/p7.java
class Demo{
	void fun(int num){
		if(num<=1)
			return;
		fun(num-2);
		System.out.println(num);
		fun(num-1);
	}
	public static void main(String[]args){
		Demo obj =new Demo();
		obj.fun(5);
	}
}

